custom_components/google_home/manifest.json

```diff
@@ -7,7 +7,7 @@
   "domain": "google_home",
   "issue_tracker": "https://github.com/leikoilja/ha-google-home/issues",
   "name": "Google Home",
-  "requirements": ["glocaltokens==0.2.8"],
-  "version": "1.0.1",
+  "requirements": ["glocaltokens==0.3.0"],
+  "version": "1.2.0",
```

Comment:
This should be updated on new version release.

Reply:
Yeah, but I think this should be in a separate PR with github release on top of it.